### ISSUE TITLE
False negative in UnnecessarySemicolonInEnumeration for multiple semicolons in enum body

### ISSUE THREAD

**Issue by ZaheerAhmadDev** [checkstyle/checkstyle#20470]
I have read check documentation: https://checkstyle.org/checks/coding/unnecessarysemicoloninenumeration.html
I have downloaded the latest checkstyle from: https://checkstyle.org/cmdline.html#Download_and_Run
I have executed the cli and showed it below, as cli describes the problem better than 1,000 words

```
 ~/Desktop/check
$ javac src/test.java

 ~/Desktop/check 
```
```
$ cat config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
  "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
  "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">
  <module name="TreeWalker">
    <module name="UnnecessarySemicolonInEnumeration"/>
  </module>
</module>
```

```java
$ cat src/test.java
enum Test {
    A,b,; // should show violation
    ; // should show violation
}
```
```
$ java  -jar checkstyle-13.6.0-all.jar -c config.xml src/Test.java
Starting audit...
Audit done.
```

**Describe what you expect in detail.**
From check description : 
> Semicolon is not needed if enum body contains only enum constants.

Check need to show violation for both semicolon `;`.Both semicolon are unnecessary as per check description so
both should be flagged.

**Comment by romani** [comment]
thanks a lot !!
please send PR.
